sympy/solvers/inequalities.py

```diff
@@ -489,8 +488,6 @@ def valid(x):
 
 
 def _solve_inequality(ie, s):
-    """ A hacky replacement for solve, since the latter only works for
```

Comment:
Please don't remove this. When it's not hacky it will be removed. In the meanwhile, it's the only docstring we have for the existence of this function.